stock_scanner/__manifest__.py

```diff
@@ -0,0 +1,48 @@
+# © 2011-2015 Sylvain Garancher <sylvain.garancher@syleam.fr>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Stock Scanner',
+    'summary': 'Allows managing barcode readers with simple scenarios',
+    'version': '12.0.1.0.0',
+    'category': 'Generic Modules/Inventory Control',
+    'website': 'https://odoo-community.org/',
```

Comment:
website should be: https://github.com/OCA/stock-logistics-barcode

Reply:
done